### DISCUSSION TITLE
Is it possible to get each char of text like shape object with points?

### DISCUSSION THREAD

**Discussion by Dotsen** [pymupdf/PyMuPDF#3857] [Looking for help]
If we convert PDF to SVG, we get all text as PATH elements with shapes, but each character separate. We want to read PDF as text and with some methods get shape objects with points of each letter. Is it possible? Thanks!

**Comment by JorjMcKie** [comment]
You can convert to SVG with extractable tex via `page.get_svg_image(text_as_path=False)`.

**Reply by Dotsen** [reply]
Thank you for reply.
Yes, it works, but I need both scenario in one solution.
To get text and their shape with points.
